APIURLS.md

```diff
@@ -0,0 +1,110 @@
+Admin Features:
+    Manage Project:
+        Projects Database:
+            localhost:8080/API/v1/project/id/1
+            [Post, Put, Get, Delete]
+        Create Project:
+            POST localhost:8080/API/v1/projects/id
+        Search Project:
+            GET localhost:8080/API/v1/projects?search=id
+        Delete Project:
+            Delete localhost:8080/API/v1/projects/id
+        List Project Table:
+            GET localhost:8080/API/v1/projects
+    User Management:
+        Users Database:
+            localhost:8080/API/v1/User/id/1
+            [Post, Put, Get, Delete]
+            localhost:8080/API/v1/Users
+        Invite Member: (send user invitation by email)
+            POST localhost:8080/API/v1/projects/id/Users/id

```

Comment:
Send user invitation by email: POST localhost:8080/API/v1/Users/invite
Add the invited member: POST localhost:8080/API/v1/projects/id/members/id

Reply:
What is the post request and response? Basically they should be in json format.
@Lambton42 @ChunhuiY @zoeee06

Reply:
The request of this specific POST URL should be:

POST /API/v1/Users/invite HTTP/1.1
Host: localhost:8080
Content-Type: application/json

{
  "email": "User@example.com",
  "projectId": "id",
  "inviterId": "id",
}

and the Response of this URL should be:

{
  "success": true,
  "message": "Invitation sent successfully.",
  "projectDetails": {
    "projectId": "id",
    "projectName": "exampleName",
  },
  "invitedUser": {
    "email": "User@example.com",
    "inviterId": "id",
    "status": "pending"
  }
}